assets/content-popup/app/components/widget-list.js

```diff
@@ -43,6 +43,8 @@ export default Vue.component('widget-list', {
       widgetTitleSearch: '',
       showMoreBtn: true,
       hasData: true,
+      selectedDraftWidget: false,
+      selectedDraftWidgetEditUrl: ''
```

Comment:
widgets do not have a special draft edit url, so suggest we just use `editUrl`

Reply:
its selected draft widget edit url, isn't it convincing ?

Reply:
I mean it could be smth. like `selectedWidget: ...` and than `if selectedWidget.isDraft` or `selectedWidget.editUrl`, e.g. just one variable instead of 2 (or more if there will be more use-cases)